app/services/job_runner/health_checker.rb

```diff
@@ -0,0 +1,30 @@
+module JobRunner
+  class HealthChecker
+    Summary = Struct.new(:healthy, :result) do
+      def as_json(*args)
+        to_h.as_json(*args)
+      end
+
+      alias_method :healthy?, :healthy
+    end
+
+    class << self
+      def check
+        result = Runner.configurations.map do |job_configuration|
+          [job_configuration.name, successful_job_run_within_2_intervals?(job_configuration)]
+        end.to_h
+        healthy = !result.value?(false)
+        Summary.new(healthy, result)
+      end
+
+      private
+
+      def successful_job_run_within_2_intervals?(job_configuration)
```

Comment:
This should probably return false if the job has an error.

Reply:
Maybe leave a comment to that effect?

Reply:
It does seem possible that we'll find this annoying and want to go back to considering the status healthy if all jobs at least attempted to run, even if some of them are persistently failing. Depends on how good the notice_error around individual job failures proves to be at giving us notice of failures.

But I think it's good with `error: nil` for now.